src/@types/files/index.ts

```diff
@@ -0,0 +1,5 @@
+export interface SharedItem {
```

Comment:
i would move this to src/@types/intents/index.ts

Reply:
attempted fix